packages/env/src/index.ts

```diff
@@ -33,6 +33,23 @@ export function createSHA256(): SHA256 {
   };
 }
 
+export function randomUUIDFromString(input: string) {
+  // Create a hash using SHA-1
+  const hash = createHash("sha1").update(input).digest("hex");
```

Comment:
```suggestion
  const hash = createSHA256().update(input).digest("hex");
```
check if this still generates a UUID and add test cases